### DISCUSSION TITLE
Any customized version of Bao to run in RISC-V platforms without H-extension support ?

### DISCUSSION THREAD

**Discussion by yli147** [bao-project/bao-hypervisor#138] [General]
Is there any customized version of Bao to run in RISC-V platforms without H-extension support ?

**Comment by danielRep** [comment]
Hi @yli147 , sorry for the delayed reply. 
The quick answer is no, there isn't a Bao version for RISC-V w/o the H-extension. Any particular reason you ask @yli147 ?

PS: converting this to a discussion

**Reply by yli147** [reply]
Never mind, thanks for the reply. 
The reason I asked this question is that most existing RISC-V h/w do not have h-extension.

By the way in this paper https://www.researchgate.net/publication/362293125_HSP-V_Holistic_Static_Partitioning_on_RISC-V_Platforms,  Sousa also mentioned one embedded approach by using OpenSBI domain, that's why I curiously whether Bao has a version without h-extension.
Or any suggestion on this ?
